packages/model/src/type-system.ts

```diff
@@ -5,11 +5,97 @@ import { Result } from './result'
 import { LazyTypeWrapper } from './unsafe'
 import { Expand } from '@mondrian-framework/utils'
 
+/**
+ * A type that can be defined with the Mondrian framework.
+ * 
+ * To learn more you can read about [the Mondrian model.](https://twinlogix.github.io/mondrian-framework/docs/docs/model)
+ * 
+ * ## Examples
+ * 
+ * The library function `integer()` can be used to define the model for an
+ * integer type:
+ * 
+ * ```ts
+ * const intType : Type = integer()
```

Comment:
diciamo che esplicitare il tipo Type solitamente è un errore perché in questo caso
```
type intType = Infer<typeof intType> //infer unknown
```
quindi magari toglierei l'esempio per non fuorviare un utilizzatore

PS: c'è anche AnyType che è simile ma anziché l'interfaccia base è l'unione di qualsiasi tipo

Reply:
Ok allora lo tolgo 👍